src/Command/ProcessTransferCommand.php

```diff
@@ -105,34 +105,49 @@ private function createTransfers(array $miraklOrders): void
             return;
         }
 
-        $toCreateIds = array_column($miraklOrders, 'order_id');
-        $alreadyCreatedMiraklIds = $this->stripeTransferRepository->findAlreadyCreatedMiraklIds($toCreateIds);
-        $ignoreOrderLineStates = array('REFUSED', 'CANCELED');
+        $ignoredStripeTransferStatuses = array(StripeTransfer::TRANSFER_CREATED);
+        $ignoredOrderLineStates = array('REFUSED', 'CANCELED');
+
+        $existingStripeTransfers = $this->stripeTransferRepository->findBy([
+            'miraklId' => array_column($miraklOrders, 'order_id'),
+            'type' => StripeTransfer::TRANSFER_ORDER,
+        ]);
+        $existingStripeTransfersMiraklIds = array_map(
+            function (StripeTransfer $relatedStripeTransfer) {
+                return $relatedStripeTransfer->getMiraklId();
+            },
+            $existingStripeTransfers
+        );
```

Comment:
je ferais plutôt une map :
```suggestion
        $existingStripeTransfersMiraklIds = [];
        foreach($existingStripeTransfers as $transfer) {
            $existingStripeTransfersMiraklIds[$transfer->getMiraklId()] = $transfer;
        }
```

Comme ça dessous tu peux faire 
`if(array_key_exists($miraklOrder['order_id'], existingStripeTransfersMiraklIds) { ... }` et récupérer le transfer directement.